tests/pytorch/r2.2/common.libsonnet

```diff
@@ -19,14 +19,14 @@ local utils = import 'templates/utils.libsonnet';
 local volumes = import 'templates/volumes.libsonnet';
 
 {
-  local r2_1 = {
-    frameworkPrefix: 'pt-2.1',
+  local r2_2 = {
+    frameworkPrefix: 'pt-2.2',
     tpuSettings+: {
-      softwareVersion: 'pytorch-2.1',
+      softwareVersion: 'pytorch-2.2',
     },
-    imageTag: 'r2.1.0_3.8',
+    imageTag: '2.2.0',
```

Comment:
Not sure if this imageTag refers to existing torch_xla image tag, or image will be built in the xl ml test workflow. Let's verify with a oneshot script after the 2.2 whl is ready.

Reply:
cc @will-cromar @gkroiz in case you guys have insight on the `imageTag` field here.

Reply:
I believe this imageTag should refer to an existing image tag, xl ml test workflow won't produce a new pytorch image.

Reply:
This tag should match the pattern of the release docker image, such as `r2.2.0-rc1_3.10` in this case. FYI, this is only relevant to the GPU tests right now, since the TPU VM tests don't use a docker image.